exercises/two-fer/introduction.md

```diff
@@ -5,4 +5,4 @@ Two-for-one is a way of saying that if you buy one, you also get one for free.
 So the phrase "two-fer" often implies a two-for-one offer.
 
 Imagine a bakery that has a holiday offer where you can buy two cookies for the price of one ("two-fer one!").
-You go for the offer and (very generously) decide to give the extra cookie to a friend.
+You take the offer and (very generously) decide to give the extra cookie to someone else in the queue.
```

Comment:
Should this be, "You accept the offer"?

Reply:
It could be, colloquially it means the same thing.

Reply:
Given that we have people for whom English is not a first language, should we try to avoid colloquialisms?

Reply:
Colloquially does not necessarily mean hard to understand, it only means informal, and we use informal language here.  It is conversational.  If it were slang, then it would be less appropriate.

We do not currently avoid colloquialisms.